basic-sql-commands.md

```diff
@@ -194,16 +194,73 @@ INSERT INTO table1 (column_d, column_c, column_b, column_a) VALUES
     ('aaaa', DATE '2016-01-01', DEFAULT, 12345);
 ```
 ```
-+----------+----------+------------+----------+
-| column_a | column_b |  column_c  | column_d |
-+----------+----------+------------+----------+
-| 12345    |     NULL | 2016-01-01 | aaaa     |
-+----------+----------+------------+----------+
++----------+----------+---------------------------------+----------+
+| column_a | column_b |            column_c             | column_d |
++----------+----------+---------------------------------+----------+
+|    12345 |     NULL | 2016-01-01 00:00:00 +0000 +0000 | "aaaa"   |
++----------+----------+---------------------------------+----------+
 ```
 
 ## Query a Table
 
+To query a table, use the `SELECT` command followed by the columns to be returned and the table from which to retrieve the data:
+
+```postgres
+SELECT column_a, column_b FROM table1;
+```
+```
++----------+----------+
+| column_a | column_b |
++----------+----------+
+|    12345 |      1.1 |
+|    54321 |      1.2 |
+|    67890 |      1.3 |
++----------+----------+
+```
+
+To retrieve all columns, use the `*` wildcard:
+
+```postgres
+SELECT * FROM table1;
+```
+```
++----------+----------+---------------------------------+----------+
+| column_a | column_b |            column_c             | column_d |
++----------+----------+---------------------------------+----------+
+|    12345 |      1.1 | 2016-01-01 00:00:00 +0000 +0000 | "aaaa"   |
+|    54321 |      1.2 | 2016-01-01 00:00:00 +0000 +0000 | "bbbb"   |
+|    67890 |      1.3 | 2016-01-02 00:00:00 +0000 +0000 | "cccc"   |
++----------+----------+---------------------------------+----------+
+```
+
+To filter the results, add a `WHERE` clause identifying the columns and values to filter on: 
+
+```postgres
+SELECT column_a, column_b FROM table1 WHERE column_b < 1.3;
+```
+```
++----------+----------+
+| column_a | column_b |
++----------+----------+
+|    12345 |      1.1 |
+|    54321 |      1.2 |
++----------+----------+
+```
+
+To sort the results, add an `ORDER BY` clause identifying the columns to sort by and whether the sort should be ascending or descending:
```

Comment:
This implies the `DESC` indicator is per clause, but it is per column. Maybe "identifying the columns to sort by. Optionally, an ascending or descending direction indicator can be used for any column(s)."